api_types.ts

```diff
@@ -382,6 +382,7 @@ export interface ParamDef<T extends UnionType> {
   allowManualInput?: boolean;
 
   // TODO(patrick): Unhide this
+  /** @deprecated use `ParentDefinition` instead */
```

Comment:
This won't be considered deprecated anymore.

Reply:
Done.